packages/notched-outline/index.tsx

```diff
@@ -122,6 +121,7 @@ export default class NotchedOutline extends React.Component<
         <div
           className={this.classes}
           key='notched-outline'
+          style={ {width: this.state.actualWidth ? `${this.state.actualWidth}px`: undefined} }
```

Comment:
The render method needs to change quite a bit. I think with the new notchedoutline, it requires 3 divs